lib/aur-sync

```diff
@@ -192,6 +201,10 @@ while true; do
         --nover|--no-ver)
             chkver=0
             shift ;;
+        --nover-shallow|--no-ver-shallow)
+            chkver=1
```

Comment:
`chkver` is `1` by default anyway

Reply:
Yeah, but it leaves out any ambiguity this way. 

In general, I've been wondering if we should do like `git` and have positive flags, even for options that are enabled by default...